neutron/neutron.go

```diff
@@ -0,0 +1,372 @@
+// goose/neutron - Go package to interact with OpenStack Network Service (Neutron) API V2.0.
+// See documentation at:
+// http://developer.openstack.org/api-ref/networking/v2/index.html
+
+package neutron
+
+import (
+	"fmt"
+	"gopkg.in/goose.v1/client"
+	"gopkg.in/goose.v1/errors"
+	goosehttp "gopkg.in/goose.v1/http"
+	"net/http"
+)
+
+const (
+	ApiFloatingIPsV2        = "v2.0/floatingips"
```

Comment:
The "service URL" is whatever endpoint is registered in keystone, which it seems is always for a specific API version.